### DISCUSSION TITLE
How to package a genie project from windows environment to linux server？

### DISCUSSION THREAD

**Discussion by ck-domineer** [GenieFramework/GenieFramework.jl#13] [Q&A]
How to package a genie project from windows environment to linux server？

**Comment by essenciary** [comment]
Genie apps are not os dependent so you can use any linux deployment method. For example you could use docker if you're familiar with it and your hosting provider supports it, or you could manually deploy the app and configure it with a web server like nginx. We have documentation for both of these. Can you provide more details about what you need? FYI @PGimenez

**Reply by ck-domineer** [reply]
Thanks for your suggestion. I need to create a binary to pass my genie project without going through code, I tried using PackageCompiler to do the job but it doesn't seem very feasible, so I wanna ask for help if anyone has similar experience to create a binary program for genie project.

**Reply by PGimenez** [reply]
As Adrian mentioned,our recommended way is to package the app into a Docker container. You can find more details on this in our docs

https://learn.genieframework.com/framework/guides/deployments/deploying-genie-apps

As for compiling...I've tried using PackageCompiler once before and wasn't successful. I'll try again within the following days for a project I'm working on; I'll update you with the outcome.

**Reply by ck-domineer** [reply]
OK, thanks for your help! We can not use docker either. I will continue to try to complete the compilation of our project. We also look forward to your good news at the same time.
